frictionless/actions/extract.py

```diff
@@ -6,7 +6,9 @@
 from .. import errors
 
 
-def extract(source=None, *, type=None, process=None, stream=False, **options):
+def extract(
+    source=None, *, type=None, process=None, stream=False, filter_func=None, **options
```

Comment:
Is it possible to name this arg `filter` instead of `filter_funct`? I think Python will allow it.

We ca use then:

```python
import builtins
data = builtins.filter(filter, data) if filter_func else data
```